packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -98,6 +99,10 @@ - (void)addObservers:(AVPlayerItem *)item {
          forKeyPath:@"playbackBufferFull"
             options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
             context:playbackBufferFullContext];
+  [player addObserver:self
+           forKeyPath:@"rate"
+              options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
+              context:rateContext];
```

Comment:
interesting that this is observing the player, which is different from the rest - does `item` provide `rate` key path as well?

Reply:
I have added comments